UC-Berkeley-Extension/global
# Set preprocess_block variables to be used in custom blocks

## Description
Made node data available to be used inside blocks when on program nodes:
- `nid`
- `type`
- `title`
- `shortname`
- `link`
- `certType`

And set the hero to auto-populate using the node values.

Used the same system to make the program name available inside the comparison card block.

## Testing instructions
1. Pull branch and `drush cim`
1. Verify that Data Science hero shows the correct values.
1. Change some values on the Data Science hero and verify it doesn't show up when viewing the page.
1. Verify hero text shows up on the home page.
1. Create a new program and verify the values show up on the program hero.
